constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/Placeholder.java

```diff
@@ -120,22 +123,20 @@ public View getContent() {
   public void onDraw(Canvas canvas) {
     if (isInEditMode()) {
       canvas.drawRGB(223, 223, 223);
-      Paint paint = new Paint();
-      paint.setARGB(255, 210, 210, 210);
-      paint.setTextAlign(Paint.Align.CENTER);
-      paint.setTypeface(Typeface.create(Typeface.DEFAULT, Typeface.NORMAL));
-
-      Rect r = new Rect();
-      canvas.getClipBounds(r);
-      paint.setTextSize(r.height());
-      int cHeight = r.height();
-      int cWidth = r.width();
-      paint.setTextAlign(Paint.Align.LEFT);
+      textPaint.setARGB(255, 210, 210, 210);
+      textPaint.setTextAlign(Paint.Align.CENTER);
```

Comment:
I'm not sure, but maybe we could set it to `LEFT` directly, and move it to the constructor?

Reply:
I'll make that change but also now realize I should have suppressed this check as its conditionally only in layout preview.